packages/dev/s2-docs/src/PropTable.tsx

```diff
@@ -162,12 +162,21 @@ function Rows({props, showDefault, showRequired}: {props: TInterface['properties
   return properties.map((prop, index) => (
     <React.Fragment key={index}>
       <TableRow>
-        <TableCell role="rowheader" hideBorder={!!prop.description}>
+        <TableCell role="rowheader" hideBorder={!!prop.description} styles={style({whiteSpace: 'nowrap'})}>
```

Comment:
at smaller screen sizes, maybe it should be allowed to break? it was just bad on desktop

Reply:
On mobile the columns are stacked vertically so it's less of an issue.